pkg/daemon/ceph/client/mirror.go

```diff
@@ -376,6 +376,12 @@ func EnableRBDRadosNamespaceMirroring(context *clusterd.Context, clusterInfo *Cl
 		return errors.Errorf("ceph version %q does not support mirroring in rados namespace %q with --remote-namespace flag, supported version are v20 and above.", clusterInfo.CephVersion.String(), poolAndRadosNamespaceName)
 	}
 
+	implicitnsKey := "<implicit>"
+	implicitnsVal := ""
+	if remoteNamespace == &implicitnsKey {
+		remoteNamespace = &implicitnsVal
+	}
+
```

Comment:
@subhamkrai do we still push the mirroring status if the mode is `init-only` on the blockpool CR? wont that be wrong? because the mirroring status now will be on the radosnamespace CR?

Reply:
@Madhu-1 are you suggestion in `init-only` mode mirroring status should be in radosnamespace CR only and not in blockpool CR?

Reply:
Yes because in init only no images will get mirrored and the mirroring will be enabled in the Radosnamespace CR's not at the pool level. it make sense to not have mirroring status in init-only mode

Reply:
got it

Reply:
done